quinn-proto/src/packet.rs

```diff
@@ -30,16 +30,35 @@ pub struct PartialDecode {
 
 #[allow(clippy::len_without_is_empty)]
 impl PartialDecode {
-    #[allow(unreachable_pub)] // fuzzing only
+    /// Create an instance of [`PartialDecode`], expecting a fixed-length local connection id
```

Comment:
If we're making this public, we should document the return type here and below. In particular, it's not obvious what the `Option<BytesMut>` is.

Reply:
This doesn't seem to have been addressed?